nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Reactivate deactivated users

# Description
Created a "Deactivated Users" view to show deactivated users. From that view, deactivated users can be reactivated. I also changed the trash can icons in this view to power buttons, to better reflect the intent of deactivation/reactivation.
Fixes #27 
## Type of change
# Testing Instructions
1. Run the app through the VS debugger or ```dotnet run```
2. Navigate to Admin Tools > User Profiles
3. If no users have been deactivated, go ahead and deactivate some
4. Click the "Deactivated Users" button to view deactivated users
5. Click the green power button to reactivate the user
6. A confirmation window will make sure you wish to reactivate, then take you back to the deactivated users list
# Checklist:
